### PR TITLE
Allow optional uri to support https://crm.zoho.eu clients

### PR DESCRIPTION
**Pull request by ghost**
Hi

First off thanks for this very useful library.  When we tried to use it for EU based clients we noticed that the URL had been hard coded  to https://crm.zoho.com whereas clients in the EU are hosted on https://crm.zoho.eu.  So I've added an optional parameter to the constructor which defaults to https://crm.zoho.com to provide backward compatibility.  If you are happy with the approach I'd appreciate a merge to reduce the number of forks we have!

Regards

Steven Brookes

### CODE REVIEW

File: src/Client.php [hunk 1]
```diff
@@ -29,7 +31,7 @@ public function call($module, $method, $params = array(), $data = array())
 
         // Set basic options
         $options = array(
-            'base_uri' => 'https://crm.zoho.com/crm/private/json/',
+            'base_uri' => $this->uri.'/crm/private/json/', #https://crm.zoho.com or https://crm.zoho.eu
```
Comment (maidmaid) [inline src/Client.php]: Could you replace the shell-style comment `#` by `//`.

**Review by maidmaid — CHANGES_REQUESTED** [review]

**Comment by maidmaid** [comment]
Hello @talisman-innovations, and thank you for this contribution. I didn't know that the URI could change! Just have a look of my review, and then I'll merge your PR as soon as that will be fixed.

**Comment by talisman-innovations** [comment]
Hi @maidmaid   - updated as requested.
Regards
Steve

**Review by maidmaid — APPROVED** [review]

**Comment by maidmaid** [comment]
Thank you @talisman-innovations.

**Pull request merged**